src/Core/Utility/Win32.re

```diff
@@ -0,0 +1 @@
+external freeConsole: unit => unit = "win32_free_console";
```

Comment:
One thing I noticed is that the `bindings.c` that implements this function should also be moved to `Core/Utility` - and the `(c_names bindings)` moved from `Core/dune` to `Core/Utility/dune`. But this unfortunately doesn't address the windows build error

Reply:
Good catch! Fixed in https://github.com/onivim/oni2/pull/1220/commits/5001690541b9a04cb040a107fa0a9c640a698083

I find it a bit uncomfortable how little encapsulation there are with dependencies though, even more so with c dependencies.